### PR TITLE
news: GRASS GIS 7.8.3

### PR DESCRIPTION
**Pull request by neteler**
TODO: add catchy screenshot (anyone?!)

### CODE REVIEW

**Comment by veroandreo** [comment]
Looks good to me :)

One question regarding the 7.8.3-News page: if `Actions` was set for master, is it ok to put it in this release or should we better keep that for future grass 8 release news?

**Comment by neteler** [comment]
> Looks good to me :)
> 
> One question regarding the 7.8.3-News page: if `Actions` was set for master, is it ok to put it in this release or should we better keep that for future grass 8 release news?

AFAIK @wenzeslaus is working on a reduced GH `Actions`  setup also for relbranch78, correct?

**Comment by neteler** [comment]
TODO add this entry: 

the 7.8.3 is now available for mac on MacPorts:  https://ports.macports.org/port/grass7/summary

File: content/news/2020_05_05_GRASS_GIS_7_8_3_released.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+-   winGRASS 7.8.3: 
+    [32bit standalone installer](/grass78/binary/mswindows/native/x86/WinGRASS-7.8.3-1-Setup-x86.exe) \|  [64bit standalone installer](/grass78/binary/mswindows/native/x86_64/WinGRASS-7.8.3-1-Setup-x86_64.exe)
+-   winGRASS 7.8.3 OSGeo4W:
+    [32bit OSGeo4W installer](http://download.osgeo.org/osgeo4w/osgeo4w-setup-x86.exe) \| [64bit OSGeo4W installer](http://download.osgeo.org/osgeo4w/osgeo4w-setup-x86_64.exe)
+-   [Debian](https://packages.debian.org/source/sid/grass)
+-   [Fedora and EPEL7](https://copr.fedorainfracloud.org/coprs/neteler/grass78/) (incl. CentOS7, RHEL7 - also included in Fedora 31+)
+-   Linux Mint: see Ubuntu
+-   [Ubuntu](https://launchpad.net/~ubuntugis/+archive/ubuntu/ubuntugis-unstable) (ubuntugis-unstable)
+-   ... further binary packages for other Linux distributions will follow shortly, please check at [software downloads](/download/software/index.html#g78x)
```
Comment (neteler) [inline content/news/2020_05_05_GRASS_GIS_7_8_3_released.md:35]:
```suggestion
- Mac: available on [MacPorts](https://ports.macports.org/port/grass7/summary)
- ... further binary packages for other Linux distributions will follow shortly, please check at [software downloads](/download/software/index.html#g78x)
```

File: content/news/2020_05_05_GRASS_GIS_7_8_3_released.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+As a follow-up to the previous GRASS GIS 7.8.2 we have published the
+**new release GRASS GIS 7.8.3**. Besides other improvements, this
+release offers numerous wxGUI improvements and a number of Mac-specific,
+more or less critical bugfixes. A notable improvement is
+the now working vector digitizer.
+
+An overview of the new features in the 7.8 release series is available
+at [new features in GRASS GIS 7.8](https://trac.osgeo.org/grass/wiki/Grass7/NewFeatures78).
+
```
Comment (neteler) [inline content/news/2020_05_05_GRASS_GIS_7_8_3_released.md:24]:
```suggestion

While still in the works, an [automated source code quality test system](https://github.com/OSGeo/grass/actions) has been started,
```

File: content/news/2020_05_05_GRASS_GIS_7_8_3_released.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+While still in the works, an [automated source code quality test system](https://github.com/OSGeo/grass/actions) has been started,
```
Comment (neteler) [inline content/news/2020_05_05_GRASS_GIS_7_8_3_released.md:25]:
```suggestion
While still in the works, an [automated source code quality test system](https://github.com/OSGeo/grass/actions) has been started.

```

File: content/news/2020_05_05_GRASS_GIS_7_8_3_released.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+
+While still in the works, an [automated source code quality test system](https://github.com/OSGeo/grass/actions) has been started.
+
+**Binaries/Installer download:**\
+
+-   winGRASS 7.8.3: 
+    [32bit standalone installer](/grass78/binary/mswindows/native/x86/WinGRASS-7.8.3-1-Setup-x86.exe) \|  [64bit standalone installer](/grass78/binary/mswindows/native/x86_64/WinGRASS-7.8.3-1-Setup-x86_64.exe)
+-   winGRASS 7.8.3 OSGeo4W:
+    [32bit OSGeo4W installer](http://download.osgeo.org/osgeo4w/osgeo4w-setup-x86.exe) \| [64bit OSGeo4W installer](http://download.osgeo.org/osgeo4w/osgeo4w-setup-x86_64.exe)
+-   [Debian](https://packages.debian.org/source/sid/grass)
```
Comment (neteler) [inline content/news/2020_05_05_GRASS_GIS_7_8_3_released.md:33]:
```suggestion
-   [Debian](https://tracker.debian.org/pkg/grass)
```

**Comment by neteler** [comment]
I'll merge without screenshot...

**Pull request merged**